contracts/interfaces/ILightClient.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
```

Comment:
for interfaces I think the practice we can follow is to leave the solidity version as generic, so that other third party contracts can import our interfaces to test/implement, without having to have all of their contracts which interact with this interface the exact same sol version as us. I think this is the general practice (e.g. used by OZ, etc)